docs/nav.js

```diff
@@ -9,14 +9,21 @@ function section(title) {
 const nav = {
     items: [
         section("Intro"),
-        page("Overview", "index", "index.md"),
+        page("Overview", "index", "index.md"), // TODO(BREAKING): For consistency & clarity, 'index' slug should be renamed 'intro'?
```

Comment:
I think that's a good rename, not sure if there's any other fallout I'm not aware/thinking of..

Reply:
it sounds like the `index` slug might be treated specially - can we avoid changing this for now?